### DISCUSSION TITLE
database app

### DISCUSSION THREAD

**Discussion by josejachuf** [poem-web/poem#247] [Q&A] [answered]
A short time ago I started with Rust. I was testing several web frameworks, I did something with Tide. A week ago I discovered poem and I liked it a lot.
I did not find, except for the official documentation, no information or blogs or youtube.
I was wondering if someone has an example of use with databases that can share.

From already thank you very much

**Comment by sunli829** [comment, marked as answer]
There are many examples [here](https://github.com/poem-web/poem/tree/master/examples), and there is a full tutorial on [`docs.rs`](https://docs.rs/poem/1.3.20/poem/).

**Comment by josejachuf** [comment]
Thanks. After I sent I saw that in examples/openapi is what I needed
